app/V1Module/presenters/UsersPresenter.php

```diff
@@ -247,6 +248,7 @@ private function changeUserPassword(?Login $login, ?string $oldPassword,
       }
 
       $login->changePassword($password);
+      $login->getUser()->setTokenValidityThreshold(new DateTime());
```

Comment:
This also revokes the token of the session in which the user changed the password. I.e., password change effectively sings user off.